File: eclipse/exe4.java
package eclipse;

import java.util.Scanner;

public class exe4 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double x = 1;
        double y = 1;

        while (x != 0 && y != 0) {
            System.out.println("Digite o valor de X: ");
            x = scanner.nextDouble();
            System.out.println("Digite o valor de Y: ");
            y = scanner.nextDouble();

            if (x > 0 && y < 0) {
                System.out.println("Quadrante A");
            } else if (x > 0 && y > 0) {
                System.out.println("Quadrante B");
            } else if (x < 0 && y > 0) {
                System.out.println("Quadrante C");
            } else if (x < 0 && y < 0) {
                System.out.println("Quadrante D");
            }
        }
        scanner.close();
    }
}

File: logic-exercises/low-level/exe4.c
#include <stdio.h>

int main(void) {
    float x = 0.1;
    float y = -0.1;

    while (x != 0 && y != 0) {
        printf("Digite o valor de X: ");
        scanf("%f", &x);
        printf("Digite o valor de Y: ");
        scanf("%f", &y);

        if (x > 0 && y < 0) {
            printf("Quadrante A\n");
        } else if (x > 0 && y > 0) {
            printf("Quadrante B\n");
        } else if (x < 0 && y > 0) {
            printf("Quadrante C\n");
        } else if (x < 0 && y < 0) {
            printf("Quadrante D\n");
        }
    }
    return 0;
}